src/main/scala/fi/oph/koski/valpas/oppija/ValpasAccessResolver.scala

```diff
@@ -59,14 +59,16 @@ class ValpasAccessResolver {
     val hasAccess = rooli match {
       case ValpasRooli.OPPILAITOS_HAKEUTUMINEN =>
         oppija.hakeutumisvalvovatOppilaitokset.nonEmpty &&
-          accessToSomeOrgs(rooli, oppija.hakeutumisvalvovatOppilaitokset)
+          accessToSomeOrgs(rooli, oppija.hakeutumisvalvovatOppilaitokset) &&
+          !oppija.oppivelvollisuudestaVapautettu
```

Comment:
En bongannut mitään automaattitestejä näille lisätyille käyttöoikeuslogiikoille. Voisi ehkä olla? Kokeilin kommentoida näitä pari pois ja ajaa ValpasOppijaLaajatTiedotServiceKäyttöoikeudetSpec , niin mikään ei feilannut.

Reply:
Näitä on fronttitesteissä